marklogic-data-hub/src/test/resources/test-projects/all-steps-referenced/flows/CurateCustomerJSON.flow.json

```diff
@@ -0,0 +1,160 @@
+{
+  "name": "CurateCustomerJSON",
+  "steps": {
+    "1": {
```

Comment:
To Srinath's point above - if we're not introducing any new assertions in the AllSteps class, you could remove the ingestion and mapping steps to simplify the test class.

Reply:
How would I get the data into the FINAL database (for matching and merging) without ingestion and mapping steps?

Reply:
In the future, you can just use HubClient to insert documents - see the ReferenceModelProject class for examples of that. 

But leaving these steps here for that purpose is fine, no need to redo.